BeatFollower/UI/EndLevelViewController.cs

```diff
@@ -1,22 +1,25 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using BeatFollower.Models;
 using BeatFollower.Services;
 using BeatSaberMarkupLanguage.Attributes;
 using BeatSaberMarkupLanguage.Components;
 using BeatSaberMarkupLanguage.ViewControllers;
+using UnityEngine;
 
 namespace BeatFollower.UI
 {
-    [HotReload(@"C:\working\BeatFollowerPlugin\BeatFollower\UI\Views\EndLevel.bsml")]
+    [HotReload(RelativePathToLayout = @"Views\\EndLevel.bsml")]
```

Comment:
Isn't a double `\` an error here, as you're taking the path as a verbatim string?

Reply:
No, I use `\\` for all of my relative path hot reloads.

Reply:
The `Path.Combine()` that's used internally probably normalizes that then.